src/routes/projects.ts

```diff
@@ -52,13 +58,26 @@ router.get('/:snootyProject/:branch/documents', async (req, res, next) => {
   const reqId = getRequestId(req);
   try {
     const metadataCursor = findLatestMetadataByProjAndBranch(snootyProject, branch, req);
-    const pagesCursor = findPagesByProjAndBranch(snootyProject, branch, req);
-    await streamData(res, pagesCursor, metadataCursor, { reqId }, req);
+
+    // base methods for no query param update
+    let pagesCursor = findPagesByProjAndBranch(snootyProject, branch, req);
+    let streamDataOptions: StreamDataOptions = { reqId };
+
+    if (req.query.updated && Number(req.query.updated)) {
+      // reconstruct to use the logic in
+      // /:snootyProject/:branch/documents/updated/:timestamp' route
+      const timestamp = req.query.updated as string;
+      const timestampNum = parseInt(timestamp);
+      pagesCursor = findUpdatedPagesByProjAndBranch(snootyProject, branch, timestampNum, req);
```

Comment:
It might be a little weird to perform two different `find` queries whenever we want updated documents. Is it possible to modify `findPagesByProjAndBranch` to conditionally support a timestamp argument? The `findPagesByProj` function does something like this. What do you think?